Jenkinsfile.release

```diff
@@ -44,9 +43,9 @@ ansiColor('xterm') {
 
               targets = ''
               if (params.dryRun == false)
-                targets = 'docker-tag linux-packages jar-artifact s3-package plugin-interface' + (params.latest ? 'docker-latest' : '')
+                targets = 'docker-tag linux-packages jar-artifact s3-package plugin-interface ' + (params.latest ? 'docker-latest' : '')
```

Comment:
don't you want to put the trailing space in front of docker-latest instead of at the end of the plugin-interface? Might be a little bit cleaner for a case when there is no param latest - although it will probably work in both cases.